edsdme/blocks/search/SearchCards.js

```diff
@@ -205,9 +205,16 @@ export default class Search extends PartnerCards {
   }
 
   generateFilters() {
+    function checkKey(key) {
```

Comment:
@SonjaPopovic - regarding the below requirement from the ticket, I think we'll have to update that in the backend anyhow, so I don't think we will need this mapping from `asset-language` to `language`. I can simply change the key in the backend. Or do you need this for something else?

```pages should not show up in the filtered view, even if they match the selected language.```

Reply:
We can also update in on backend, i prepared pr for filtering only assets https://git.corp.adobe.com/wcms/dx-partners-runtime/pull/104 , but didn't want to change key in the backend since was not sure  will that have side effects since search api is used on other pages.  On logos pages only assets are fetched anyway, and on gnav search there is no language filter so i guess it is ok. I will prepare pr as part of this ticket. Thanks

Reply:
it shouldn't have any side effects, I think the fewer mappings we keep the better